internal/modules/pip/pipenv/handler.go

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package pipenv
+
+import (
+	"errors"
+	"path/filepath"
+	"spdx-sbom-generator/internal/helper"
+	"spdx-sbom-generator/internal/models"
+	"spdx-sbom-generator/internal/modules/pip/worker"
+	"strings"
+)
+
+const cmdName = "pipenv"
+const manifestFile = "Pipfile"
+const manifestLockFile = "Pipfile.lock"
+const placeholderPkgName = "{PACKAGE}"
+
+var errDependenciesNotFound = errors.New("There are no components in the BOM. The project may not contain dependencies installed. Please install Modules before running spdx-sbom-generator, e.g.: `pipenv install` might solve the issue.")
```

Comment:
plz make error message similar to other modules

Reply:
@niravpatel27, message updated.